spec/rspec/core/configuration_options_spec.rb

```diff
@@ -422,7 +445,7 @@ def expect_parsing_to_fail_mentioning_source(source, options=[])
       File.open(File.expand_path("~/.rspec"), "w") {|f| f << "--color"}
       with_env_vars 'SPEC_OPTS' => "--example 'foo bar'" do
         options = parse_options("--drb")
-        expect(options[:color]).to be_truthy
+        expect(options[:color_mode]).to eq(:automatic)
```

Comment:
Since `--color` is deprecated, it would be nice to remove its use from examples that aren't specifically testing it.  This example is testing the various options files, and only used `--color` as an example of an option.  I'd like to see it changed to something else (perhaps `--order random` or similar?) so that when we do remove `--color` in the future we don't have to touch examples like this that are really about something else.